frontend/build-tc.sh

```diff
@@ -1,2 +1,7 @@
 #!/bin/bash
 echo ${BUILD_NUMBER:-DEV} > conf/build.txt
+
+# put the script needed on the machine into the package so it ends up on the instance via riffraff
+# the special ##teamcity command causes teamcity to write it to the build output
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+echo "##teamcity[publishArtifacts '$DIR/instance => .']"
```

Comment:
@lmath I wasn't sure the best update for the comment... maybe you can suggest a replacement?